.rubocop.yml

```diff
@@ -5,3 +5,4 @@ AllCops:
   TargetRubyVersion: 2.7
   Exclude:
     - _site/**/*
+    - 'vendor/bundle/**/*'
```

Comment:
If we need to exclude these then we can just `- vendor/**/*`

Reply:
We need to exclude these because gems are installed into this directory, see below. _vendor/bundle/_ is the only directory in _vendor/_, so I decided to be more specific and clear what exactly we exclude.

Do you think there is any reason to exclude the whole _vendor/_ at the moment?

Reply:
Yes, we should ignore the whole `vendor`. There are no files for RuboCop to analyze under `vendor`.

Reply:
There is even no `vendor/` directory yet. OK, changed.